src/HealthCheck.Core/HealthCheckResult.cs

```diff
@@ -8,14 +8,14 @@ public class HealthCheckResult
         private readonly CheckResult[] _results;
         private readonly bool _passed;
 
-        public CheckResult[] Results { get { return _results; } }
-        public bool Passed { get { return _passed; } }
-        public string Status { get { return _passed ? "success" : "failure"; } }
+        public CheckResult[] Results => _results;
+        public bool Passed => _passed;
+        public string Output => _passed ? "SUCCESS" : "FAILURE";
```

Comment:
Why the change from lower case to upper case? If someone has written a test against the healthcheck result, it would break. I don't see the value in changing this.

Reply:
Also I want the output to be as compatible as possible with [Runscope/healthcheck](https://github.com/Runscope/healthcheck).

Reply:
well.. it is just looks better in mvc page out of the box. I have added mvc module in framework whoch shows your healthcheck in simple page. So it's just for cosmetic look. but ok

Reply:
If it's for cosmetic reasons, I suggest using CSS to make the status value uppercase. That way it only affects the MVC view and not all (existing) endpoints.